src/sensor/controllers/sensor.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { SensorController } from './sensor.controller';
+import { SensorService } from '../services/sensor.service';
+
+
+describe('Sensor Controller', () => {
+  let module: TestingModule;
+  beforeAll(async () => {
+    module = await Test.createTestingModule({
+      controllers: [SensorController],
+      providers: [SensorService],
+    }).compile();
+  });
+  it('should be defined', () => {
+    const controller: SensorController = module.get<SensorController>(SensorController);
+    expect(controller).toBeDefined();
```

Comment:
To be extended for now, right? Dat is OK ... al zullen we hier wel / ook aan moeten werken om onze testen ook op orde te krijgen / brengen 😉